rkt/flag/secflags.go

```diff
@@ -23,21 +23,44 @@ const (
 	insecureOnDisk
 	insecureHTTP
 	insecurePubKey
+	insecureCapabilities
+	insecurePaths
+	insecureSeccomp
 
-	insecureAll = (insecureImage | insecureTLS | insecureOnDisk | insecureHTTP | insecurePubKey)
+	insecureAllFetch = (insecureImage | insecureTLS | insecureHTTP | insecurePubKey)
+	insecureAllRun   = (insecureOnDisk | insecureCapabilities | insecurePaths | insecureSeccomp)
+	insecureAll      = (insecureAllFetch | insecureAllRun)
 )
 
 var (
-	insecureOptions = []string{"none", "image", "tls", "ondisk", "http", "pubkey", "all"}
+	insecureOptions = []string{
+		"none",
+		"image",
+		"tls",
+		"ondisk",
+		"http",
+		"pubkey",
+		"capabilities",
+		"paths",
+		"seccomp",
+		"all-fetch",
+		"all-run",
+		"all",
+	}
 
 	insecureOptionsMap = map[string]int{
-		insecureOptions[0]: insecureNone,
-		insecureOptions[1]: insecureImage,
-		insecureOptions[2]: insecureTLS,
-		insecureOptions[3]: insecureOnDisk,
-		insecureOptions[4]: insecureHTTP,
-		insecureOptions[5]: insecurePubKey,
-		insecureOptions[6]: insecureAll,
+		insecureOptions[0]:  insecureNone,
+		insecureOptions[1]:  insecureImage,
+		insecureOptions[2]:  insecureTLS,
+		insecureOptions[3]:  insecureOnDisk,
+		insecureOptions[4]:  insecureHTTP,
+		insecureOptions[5]:  insecurePubKey,
+		insecureOptions[6]:  insecureCapabilities,
```

Comment:
FYI, I am removing `--insecure-options=seccomp` from this PR because https://github.com/coreos/rkt/pull/2753 is not merged yet.